libraries/cbor/src/values.rs

```diff
@@ -225,6 +225,12 @@ impl From<&str> for Value {
     }
 }
 
+impl From<Vec<Value>> for Value {
```

Comment:
Is there specific reasons we don't have this before? I need to auto parse `Vec<Value>` in cbor_map! macro. If we have `From<Vec<(Value, Value)>>` for map I think we can have this for array too.